### PR TITLE
add event data export documentation, add chart image, hide introduction

### PR DESCRIPTION
**Pull request by mauerbac**
@tscanlin @krishnan-chandra

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -29,17 +29,17 @@ var BASE_NAV_SECTIONS = {
 
 
 var DATA_NAV_SECTIONS = {
-  'introduction': {
-    path: 'introduction',
-    title: 'Introduction',
-  },
+ // 'introduction': {
```
Comment (tscanlin) [inline config.js]: If we don't need need this can it be deleted?

File: src/content/data/export/index.md [hunk 2]
```diff
@@ -1,16 +1,37 @@
...
 
-*Welcome! This page walks you through everything for Raw Data Export*
 
-## Intro 
+## Intro
 
-TBA
+Event Data Export allows developers to access all of their Optimizely event data. This data is computed daily and contain the last 24 hours of events for all A/B testing experiments in an account. The data will be written securely to an S3 bucket, which you can then programmatically access via Amazon’s APIs and a set of secure credentials provided by Optimizely.
+
+You can use Raw Data Export to integrate Optimizely experiment data with your own data warehouse.
```
Comment (tscanlin) [inline src/content/data/export/index.md]: Should this say "Event Data Export" instead of "Raw Data"?

Comment (mauerbac) [inline src/content/data/export/index.md]: Yes! Nice catch.

**Comment by tscanlin** [comment]
Some small feedback, but this looks good to me!

File: config.js (hunk 1 above)
Comment (mauerbac) [inline config.js]: I plan to add an intro after Event API is finished, so just hiding for the time being.

**Comment by mauerbac** [comment]
Thanks @tscanlin . How does this look?

**Comment by tscanlin** [comment]
LGTM!

**Comment by mauerbac** [comment]
Merged! CC @krishnan-chandra

**Pull request merged**